extensions/skyportal/static/js/ducks/alert.js

```diff
@@ -0,0 +1,84 @@
+import * as API from '../API';
+import store from '../store';
+
+
+export const FETCH_ALERT = 'skyportal/FETCH_ALERT';
+export const FETCH_ALERT_OK = 'skyportal/FETCH_ALERT_OK';
+export const FETCH_ALERT_ERROR = 'skyportal/FETCH_ALERT_ERROR';
+export const FETCH_ALERT_FAIL = 'skyportal/FETCH_ALERT_FAIL';
+
+export const FETCH_AUX = 'skyportal/FETCH_AUX';
+export const FETCH_AUX_OK = 'skyportal/FETCH_AUX_OK';
+export const FETCH_AUX_ERROR = 'skyportal/FETCH_AUX_ERROR';
+export const FETCH_AUX_FAIL = 'skyportal/FETCH_AUX_FAIL';
+
+// export const FETCH_CUTOUT = 'skyportal/FETCH_CUTOUT';
+// export const FETCH_CUTOUT_OK = 'skyportal/FETCH_CUTOUT_OK';
+// export const FETCH_CUTOUT_ERROR = 'skyportal/FETCH_CUTOUT_ERROR';
+// export const FETCH_CUTOUT_FAIL = 'skyportal/FETCH_CUTOUT_FAIL';
+
+export function fetchAlertData(id) {
+    return API.GET(`/api/alerts/ztf/${id}`, FETCH_ALERT)
```

Comment:
Is there a way to implement this without having `ztf` hardcoded in? Otherwise, we will have to reimplement separate action dispatch functions for Gattini, LSST, etc.

Reply:
Yea, I know. I actually first implemented this without hardcoding ZTF and then re-did it this way since I thought people might appreciate the "explicitness". 
What would be the alternative? Resolving the alert stream by ${id} (e.g. startswith "ZTF")?